pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -262,6 +263,11 @@ func TestToKubeContainerStatusWithResources(t *testing.T) {
 						MemoryLimitInBytes: 524288000,
 						OomScoreAdj:        -998,
 					},
+					Windows: &runtimeapi.WindowsContainerResources{
```

Comment:
What is this test hoping to achieve? The current implementation for windows returns nil.

https://github.com/kubernetes/kubernetes/blob/20fc8a939c3394bd31edd06a893b75d1c312e27c/pkg/kubelet/kuberuntime/kuberuntime_container_windows.go#L136-L139

Should we not add functional code there and then add tests? Also, in practice we will have either `LinuxContainerResources` or `WindowsContainerResources`, not both.